stablehlo/reference/Scope.cpp

```diff
@@ -28,14 +28,15 @@ void Scope::add(Value ssaValue, Tensor runtimeValue) {
   if (stack_frame_.count(ssaValue))
     llvm::report_fatal_error("Duplicate SSA register found in scope");
 
-  if (ssaValue.getType() != runtimeValue.getType())
-    llvm::report_fatal_error(
-        "Expected same type for an SSA register and its evaluated value");
+  // if (ssaValue.getType() != runtimeValue.getType())
```

Comment:
Do we need to comment them out?

Reply:
Artifact from debugging :) thanks for catching!